pkgs/development/python-modules/amaranth-boards/default.nix

```diff
@@ -20,8 +20,8 @@ buildPythonPackage rec {
     hash = "sha256-dwZCKMJnEY9RjzkcJ9r3TEC7W+Wfi/P7Hjl4/d60/qo=";
   };
 
-  nativeBuildInputs = [ setuptools-scm ];
-  propagatedBuildInputs = [ setuptools amaranth ];
+  nativeBuildInputs = [ setuptools setuptools-scm ];
+  dependencies = [ setuptools amaranth ];
```

Comment:
as i understand, `nativeBuildInputs` adds the /bin paths to `PATH` while `buildInputs` adds /lib and /include to their respective environment variables.
If the package is built in a cross setup, we want to use the build variant of setuptools, but for propagation we want the target host variant.

Reply:
Sorry for the delay, i went AFK. I'll go ahead and merge it now, since i want to address the merge conflict that will arise in an other PR when i do so.